lang/java/src/main/java/com/google/devtools/build/bfg/BUILD

```diff
@@ -30,10 +30,6 @@ java_library(
         "//thirdparty/jvm/com/google/guava",
         "//thirdparty/jvm/org/eclipse/jdt:org_eclipse_jdt_core",
     ],
+    visibility = ["//lang:__subpackages__"],
```

Comment:
@cgrushko is this the appropriate visibility for the `JavaSourceFileParser`? 

For context, I moved the `JavaSourceFileParserCli` binary to the top of the directory. This was so one simply needed to type 
```
//src:JavaSourceFileParserCli
``` 
rather than the more tedious
```
//src/main/java/com/google/devtools/build/bfg:JavaSourceFileParserCli
```
The previous approach was annoyingly tedious. 

As far as the visibility, I want to expose `JavaSourceFileParser` to _just_ `JavaSourceFileParserCli` and its subpackages. However, that introduces a circular dependencies.

Reply:
Yeah, you can't set visibility to a single rule.

How often do you expect users to build the thing? 
When I built it it was from an IDE so didn't have to type it at all. 
To run it, I had a symlink, but this arguably raises the bar for contributions.

We can add an `alias()` to lang/java/BUILD that points at the full path, but tbh this seems like dead weight.